### ISSUE TITLE
Spotting Highly Important Requests and Alerting Rescue Team heads

### ISSUE THREAD

**Issue by hari03** [IEEEKeralaSection/rescuekerala#246]
Request for help is being populated from various sources and listed in the site. One major source is this [KeralaFloods2018 FB Page Help Requests Sheet](https://docs.google.com/spreadsheets/d/1sBOM5sZB3M60Jiz-SXw51ZZmlQaOhRyiZiitIcPAr7s/edit#gid=2092866803). 

There is a request made by father asking to help his daughter who is suffering from labour pain , few minutes before. We need to append our tables with the data from these excel sheets, look for such keywords and alert the Rescue Head of that area to arrange for immediate help!

Some Important Categories to think upon and find relatable words to identify such requests :

**Pregnant women**
 Labour, labour pain, pregnant, carrying

**Babies**
n days old baby, high fever just born baby, baby, n years baby

**Old people**
n years old, death bed

**Sick People**
about to die, very sick, 

**Suffering people**
about to die, bleeding, open wound, infected

**Comment by lunaroyster** [comment]
I suggested on slack that we could figure out which help requests are more immediate by voting on them. 

One issue is that we've got multiple sources of data for volunteers to deal with. If we could gather all requests made on keralarescue.in, and the spreadsheet you mentioned, we can let people figure out which ones are the most immediate by voting upon them, and notify the rescuers in the area.
